pt/tutorials-and-examples/blog/blog.rst

```diff
@@ -1,54 +1,58 @@
-Blog Tutorial
+Construindo um blog
 #############
```

Comment:
No, the underline length should match the char length of the title.
It is harder to detect in Github editor due the editor font.
Ex: 19 characters, 19 #

Construindo um blog
###################